traveler-swift-core/TravelerKit/Delegates/WishlistFetchDelegate.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  WishlistFetchDelegate.swift
+//  TravelerKit
+//
+//  Created by Omar Padierna on 2019-09-03.
+//  Copyright © 2019 Ata Namvari. All rights reserved.
+//
+
+import Foundation
+
+/// Notified of the WishlistResult fetch
```

Comment:
+1 for the documentation

Reply:
@benRmc We document all delegates, all models, and all public methods in `Traveler`